packages/ui/src/components/Flag/FlagFilter/index.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { ReactElement, useEffect, useState } from 'react';
+import { Button, Checkbox, Space } from 'antd';
+import type { FilterConfirmProps } from 'antd/es/table/interface';
+
+import StackLayout from '../../../layout/StackLayout';
+import { FlagOption, getOptionIcon } from '../FlagDropdown';
+import { IFlagDictionary } from '../types';
+
+import styles from './index.module.css';
+
+export type TFlagFilter = {
+    dictionary?: IFlagDictionary | Record<string, never>;
+    confirm?: (param?: FilterConfirmProps) => void;
+    selectedKeys?: React.Key[];
+    setSelectedKeys?: (selectedKeys: React.Key[]) => void;
+    isClear: boolean;
+};
+
+const handleSelect = (
```

Comment:
Si tu définis ton handler dans ton composant, tu n'aurais pas besoin de rajouter selectKeys and setSelectKeys dans ses paramètres.
```
const handleSelect = (  value: string ) => {
    selectedKeys.includes(value)
        ? setSelectedKeys(selectedKeys.filter((item: React.Key) => item !== value))
        : setSelectedKeys([value, ...selectedKeys]);
};
```
Et du coup
```
<Checkbox
                            checked={selectedOption.includes(FlagOption.FLAG) ? true : false}
                            key={FlagOption.FLAG}
                            onChange={handleSelect}
                            type="checkbox"
                            value={FlagOption.FLAG}
                        >
```